robottelo/cli/user.py

```diff
@@ -93,3 +94,21 @@ def ssh_keys_info(cls, options=None):
         """
         cls.command_sub = 'ssh-keys info'
         return cls.execute(cls._construct_command(options), output_format='csv')
+
+    @classmethod
+    def personal_access_token_create(cls, options=None):
```

Comment:
Could you just call this `access_token_create`?

Reply:
Done